go-controller/pkg/ovn/ovn.go

```diff
@@ -97,7 +97,8 @@ func dialNB() (*ovsdb.OVSDB, *dbcache.Cache) {
 			Tables: map[string][]string{
 				"Logical_Switch":      {"_uuid", "name", "ports", "acls", "external_ids"},
 				"Logical_Switch_Port": {"_uuid", "name", "external_ids", "addresses", "dynamic_addresses"},
-				"ACL": {"_uuid", "external_ids"},
+				"ACL":        {"_uuid", "external_ids", "match", "action"},
+				"Port_Group": {"_uuid", "acls", "ports"},
```

Comment:
Because of this above line the code will only work for OVN 2.10 or greater. It is probably okay for now. If we want compatibility with earlier version, we likely need a if else condition.

Reply:
FIXED